### DISCUSSION TITLE
ad hoc mode in 11n

### DISCUSSION THREAD

**Discussion by rookiebeotch** [open-sdr/openwifi#261] [Q&A]
I am trying to do ad hoc mode between two zed boards. 
i can get it working but they are using 11g.

How can i make it use 11n and AMPDU?
i tried the wgd.sh 1

but not sure how to do 11n mode.

**Comment by JiaoXianjun** [comment]
I didn't try 11n before with ad-hoc.
Have you tried to search the answer on internet?
